src/kirby/components/list/list.component.scss

```diff
@@ -17,19 +16,14 @@ ul {
     p {
       margin: 0;
     }
-    
-    
 
-    &.row{
-      &:first-of-type {
-        border-top: 1px solid var(--kirby-background);
-      }
+    &.divider:not(:last-child) {
```

Comment:
Could it not be a good idea to maybe call the class for `lines` also? 

Just to keep the same terminology.

Reply:
I believe you are right - I just felt that "divider" was more saying :-) I have changed it.